projects/VRDR.Tests/Messaging_Should.cs

```diff
@@ -148,13 +148,6 @@ public void CreateSubmissionFromBundle()
             Assert.Equal(submission.StateAuxiliaryId, parsed.StateAuxiliaryId);
             Assert.Equal(submission.NCHSIdentifier, parsed.NCHSIdentifier);
         }
-
-	[Fact]
```

Comment:
Why does this test need to be removed?

Reply:
Because I had to make the `MessageBundle` property protected instead of public to prevent it from being serialized, but is no longer accessible from the test.